circleguard/investigations.py

```diff
@@ -36,6 +36,23 @@ def ur(replay, beatmap):
         diffs = [hit.error() for hit in hits]
         return np.std(diffs) * 10
 
+    @staticmethod
+    def correctedur(replay, beatmap):
```

Comment:
I'd prefer this to be implemented as an argument `corrected` in the `ur` function, defaulting to `False`, instead of an entirely new function